corehq/apps/reports/tests/test_analytics.py

```diff
@@ -58,6 +58,12 @@ def setUpClass(cls):
         super(ReportAppAnalyticsTest, cls).setUpClass()
         cls.class_setup()
 
+    @classmethod
+    def tearDownClass(cls):
+        super(ReportAppAnalyticsTest, cls).tearDownClass()
```

Comment:
`super().tearDownClass` should generally be called as the last part of a tear down because it tears down database connections. (similarly `super().setupClass` is called as the first part of the setup because it sets up database See warning example here https://docs.djangoproject.com/en/2.2/topics/testing/tools/#django.test.SimpleTestCase.allow_database_queries